### PR TITLE
Created 1.7.6.0 beta release note

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
@eternoendless & @ttoine Can you check guys, please?

(@eternoendless There are some comments to remove in the Hooks part)

Thanks

File: news/_posts/releases/2019-05-09-prestashop-1-7-6-0-beta-release.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
+---
+layout: post
+title:  "PrestaShop 1.7.6.0 beta is open for feedback!"
+subtitle: "You can download it and test it"
+date:   2019-05-09 10:45:00
```
Comment (ttoine) [inline news/_posts/releases/2019-05-09-prestashop-1-7-6-0-beta-release.md]:
```suggestion
date:   2019-05-10 12:00:00
```

File: news/_posts/releases/2019-05-09-prestashop-1-7-6-0-beta-release.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+
+![Prestashop 1.7.6.0 BETA Release](assets/images/2019/05/1.7.6_beta_banner_May19.jpg)
+
+The first Beta version for PrestaShop 1.7.6.0 is now ready for you to test!
+
+We are really happy to announce that we are close to releasing 1.7.6.0.
+This minor version (in the SemVer meaning of the term) brings important new features and improvements to the 1.7 codebase, and we can’t wait to receive your feedback!
+
+This Beta release is a great opportunity for you to start working with
```
Comment (ttoine) [inline news/_posts/releases/2019-05-09-prestashop-1-7-6-0-beta-release.md]:
```suggestion
This Beta release is a great opportunity for you to start working with it before we release a stable version. Go download and install this Beta version as a test store, either on your machine or your web hosting, then play with it – and give us your feedback! We need YOU to test this new version of PrestaShop.
```
Should be on one line to make a paragraph, then delete the lines below

File: news/_posts/releases/2019-05-09-prestashop-1-7-6-0-beta-release.md [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+it before we release a stable version. Go download and install this Beta
+version as a test store, either on your machine or your web hosting,
+then play with it – and give us your feedback! We need YOU to test this
+new version of PrestaShop.
+
+If you find regressions compared to previous versions, please create an
```
Comment (ttoine) [inline news/_posts/releases/2019-05-09-prestashop-1-7-6-0-beta-release.md]:
```suggestion
If you find regressions compared to previous versions, please create an issue on [Github](https://github.com/PrestaShop/PrestaShop/issues/new?template%3Dbug_report.md) to help us to make it stable more quickly.
```
Same.

File: news/_posts/releases/2019-05-09-prestashop-1-7-6-0-beta-release.md [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+-   Cart rules not applied if the cart contains only virtual products [\#10264](https://github.com/PrestaShop/PrestaShop/issues/10264)
+-   Hook reseted during upgrade [\#11873](https://github.com/PrestaShop/PrestaShop/issues/11873)
+-   Catalog mode correctly hides prices and cart, a new option allows
+    displaying prices while disabling orders [\#11688](https://github.com/PrestaShop/PrestaShop/pull/11688)
+-   Enhanced security checks for virtual goods [\#12023](https://github.com/PrestaShop/PrestaShop/pull/12023)
+-   The number of SQL requests to get the employee authorizations has
+    been significantly reduced [\#12914](https://github.com/PrestaShop/PrestaShop/pull/12914)
+-   Checkout doesn’t proceed automatically from carrier selection to payment selection [\#12456](https://github.com/PrestaShop/PrestaShop/issues/12456)
+
+The complete [list of bugs fixed for
```
Comment (ttoine) [inline news/_posts/releases/2019-05-09-prestashop-1-7-6-0-beta-release.md]:
```suggestion
The complete [list of bugs fixed for 1.7.6.0](https://github.com/PrestaShop/PrestaShop/issues?utf8%3D%25E2%259C%2593%26q%3Dis%253Aissue%2Bmilestone%253A1.7.6.0%2Blabel%253Afixed%2B) is available on Github.
```
one line again

File: news/_posts/releases/2019-05-09-prestashop-1-7-6-0-beta-release.md [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+## Download
+
+You can download PrestaShop 1.7.6 beta here:  
+
+[Download PrestaShop 1.7.6 beta now!](https://www.prestashop.com/en/developers-versions){: .btn .btn-lg .btn-success}
+{: .text-center}
+
+## How to help finish the 1.7.6
+
+Your help will be very appreciated to finish the work on PrestaShop
```
Comment (ttoine) [inline news/_posts/releases/2019-05-09-prestashop-1-7-6-0-beta-release.md]:
```suggestion
Your help will be very appreciated to finish the work on PrestaShop 1.7.6. Even if you are not a developer, your feedback has a lot of value.
```
one line again

File: news/_posts/releases/2019-05-09-prestashop-1-7-6-0-beta-release.md [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+value.
+
+You can:
+ - Install it and test it
+ - If you develop modules or themes, test them on this version
+ - [Report bugs on GitHub](https://github.com/PrestaShop/PrestaShop/issues) (read [how to report issues](https://devdocs.prestashop.com/1.7/contribute/contribute-reporting-issues/))
+ - Submit a bug fix by creating a [pull request](https://github.com/PrestaShop/PrestaShop/compare) (read the [contribution guidelines](https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/))
+
+ 
+If everything is working well, comment this article to tell us, and
```
Comment (ttoine) [inline news/_posts/releases/2019-05-09-prestashop-1-7-6-0-beta-release.md]:
```suggestion
If everything is working well, comment this article to tell us, and share on social networks, we also need this feedback.
```

**Review by ttoine — CHANGES_REQUESTED** [review]
Please see my comments, there are a few typo issues to fix.

File: news/_posts/releases/2019-05-09-prestashop-1-7-6-0-beta-release.md (hunk 5 above)
Comment (ghost) [reply news/_posts/releases/2019-05-09-prestashop-1-7-6-0-beta-release.md]: Did it take your reco in account

**Comment by ghost** [comment]
Know issues list up-to-date, but I only included issues related to the Core, not the one related to modules.

Don't forget to delete the "[TDC]" in the Hooks list.

**Review by ttoine — APPROVED** [review]

**Comment by ttoine** [comment]
who should delete [TBC] in the hook list? ping @eternoendless?

**Comment by ghost** [comment]
Just discussed it with Tristan, we'll add some screenshots for the main new features. I'm on it.

**Comment by ttoine** [comment]
did you add them on this pull request branch so we can see them in the preview ?

**Comment by ghost** [comment]
This time yes 🙂

**Comment by ttoine** [comment]
the preview of the article also shows now the beta banner (I included it on the branch so it's now in this pull request)

**Pull request merged**